include/tvm/node/container.h

```diff
@@ -1105,7 +1105,7 @@ class DenseMapNode : public MapNode {
   friend class MapNode;
 };
 
-#define _TVM_DISPATCH_MAP(base, var, body)    \
```

Comment:
Perhaps underscore in the macro is fine. unless there is a explict reason not to do so

Reply:
I found this in the gnu docs here http://www.gnu.org/software/libc/manual/html_node/Reserved-Names.html
`all identifiers regardless of use that begin with either two underscores or an underscore followed by a capital letter are reserved names`